config/migrations/2023/20231116175906-add-new-dienstverlenende-vereniging-s-Lim/20231116175907-add-new-dienstverlenende-vereniging-s-Lim.sparql

```diff
@@ -0,0 +1,87 @@
+PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    <http://data.lblod.info/id/bestuurseenheden/682c21df-fe6a-4266-8f97-109e5b5b4ae4>
+      rdf:type <http://data.vlaanderen.be/ns/besluit#Bestuurseenheid> ;
+      <http://mu.semte.ch/vocabularies/core/uuid>   "682c21df-fe6a-4266-8f97-109e5b5b4ae4" ;
+      skos:prefLabel "s-Lim" ;
+      <http://data.vlaanderen.be/ns/besluit#classificatie> <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/d01bb1f6-2439-4e33-9c25-1fc295de2e71> ;
+      <http://mu.semte.ch/vocabularies/ext/kbonummer> "0679631191" ;
+      <http://purl.org/dc/terms/identifier> "0679631191" ;
+      <http://mu.semte.ch/vocabularies/ext/inProvincie> <http://data.lblod.info/id/werkingsgebieden/b6868be9ca6881b1020e6cff8d431531a1cff64b39dc92646a5173b3ee144516> . # Provincie Limburg
+
+
+    <http://data.lblod.info/id/bestuursorganen/11cbdde244c6111e9937db92319339e3>
+      rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+      <http://mu.semte.ch/vocabularies/core/uuid>	"11cbdde244c6111e9937db92319339e3" ;
+      skos:prefLabel	"Directiecomité s-Lim" ;
+      <http://www.w3.org/ns/org#classification>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/5733254e-73ff-4844-8d43-7afb7ec726e8> ;
+      <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/682c21df-fe6a-4266-8f97-109e5b5b4ae4> .
+
+    <http://data.lblod.info/id/bestuursorganen/48e265168eb7d4eacf5140eb6b641468>
+      rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+      <http://mu.semte.ch/vocabularies/core/uuid>	"48e265168eb7d4eacf5140eb6b641468" ;
+      <https://data.vlaanderen.be/ns/generiek#isTijdspecialisatieVan>	<http://data.lblod.info/id/bestuursorganen/11cbdde244c6111e9937db92319339e3> ;
+      <http://data.vlaanderen.be/ns/mandaat#bindingStart>	"2019-01-01T00:00:00"^^xsd:date .
```

Comment:
`<http://www.w3.org/2001/XMLSchema#date>` is used in Loket for date types instead of `<http://www.w3.org/2001/XMLSchema#dateTime>` in Organisatie-Portaal.